lib/assessment.rb

```diff
@@ -43,6 +51,8 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    text = filename.each.gsub("\n" , " ").join(" ")
```

Comment:
`filename` is just a string, giving the path to a particular file. It is not the actual contents of the file.